modules/manage/partials/mountable-topics.adoc

```diff
@@ -11,7 +9,11 @@ An unmounted topic in object storage is detached from all clusters. The original
 
 == Unmount a topic from a cluster to object storage
 
-When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive an error message indicating that the topic is no longer available. The unmounted topic is deleted in the source cluster.
+When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive an error message indicating that the topic is no longer available. 
```

Comment:
What's the expected error message? How do I differentiate the error between an unmounted topic and one that never existed?

Reply:
@bashtanov just to check if you have those expected messages available.

Reply:
yes, it is `failed to download manifest for topic`